python/ml4ir/build/Dockerfile

```diff
@@ -9,6 +9,9 @@ ENV PYTHONPATH=/home/ml4ir/python
 
 # Install python dependencies
 COPY requirements.txt /home/ml4ir/python/
+RUN sudo apt-get --allow-releaseinfo-change update
+# Note: This is needed for DAG visualization
```

Comment:
The real question is why do we need to visualize DAGs? how more informative is it from a print?

Reply:
These are from PRs already merged to `dev_tf27` while we were working on semantic search and more complex architectures.